CoreScriptsRoot/LoadingScript.lua

```diff
@@ -36,12 +35,8 @@ end
 --
 -- Variables
 local GameAssetInfo -- loaded by InfoProvider:LoadAssets()
-local currScreenGui = nil
-local renderSteppedConnection = nil
-local fadingBackground = false
-local destroyingBackground = false
-local destroyedLoadingGui = false
-local hasReplicatedFirstElements = false
+local currScreenGui, renderSteppedConnection = nil
+local destroyingBackground, destroyedLoadingGui, hasReplicatedFirstElements = false
```

Comment:
this statement is not equivalent. in order to do this you still need to instantiate each variable by adding false, false. Otherwise only destroyingBackground gets the value false and the rest are nil